modules/contract/contracts.js

```diff
@@ -0,0 +1,246 @@
+const confFile = __dirname + '/../../config.json';
+const jsonfile = require('jsonfile');
+var eztz = {};
+var config = jsonfile.readFileSync(confFile);
+const ConseilJS = '../../lib/conseiljs';
+
+const { Helper } = require('../../helper');
+
+class Contracts{
+
+  confFile = __dirname + '/../../config.json';
+  jsonfile = require('jsonfile');
+  config = jsonfile.readFileSync(confFile);
+
+  async listContracts() {
+      await this.loadTezsterConfig();    
+      const config = this.config;
+      if(Object.keys(config.contracts).length > 0){        
+          for(var i in config.contracts){
+              console.log(Helper.output(config.contracts[i].label + " - " + config.contracts[i].pkh + " (" + config.contracts[i].identity + ")"));        
+          }
+      }
+      else{
+          console.log(Helper.outputError("No Contracts are Available !!"));        
+      }
+  }
+
+  async deployContract() {
+      var args = process.argv.slice(3);
+          if (args.length < 4) {
+              console.log(Helper.outputInfo("Incorrect usage of deploy command \nCorrect usage: - tezster deploy <contract-label> <contract-absolute-path> <init-storage-value> <account>"));
+              return;
+          }
+          await this.loadTezsterConfig(); 
+      
+          let result = await this.deploy(args[0], args[1], args[2], args[3]);
+          console.log(result);
+          console.log(Helper.outputInfo(`If you're using babylonnet node, use https://babylonnet.tzstats.com to check contract/transactions`));
+  }
+
+
+  async callContract() {
+      var args = process.argv.slice(3);
+      if (args.length < 3) {
+          console.log(Helper.outputInfo("Incorrect usage of call command \nCorrect usage: - tezster call <contract-name> <argument-value> <account>"));
+          return;
+      }
+      await this.loadTezsterConfig(); 
+      
+      let result = await this.invokeContract(args[0], args[1], args[2]);
+      console.log(result);
+      console.log(Helper.outputInfo(`If you're using babylonnet node, use https://babylonnet.tzstats.com to check contract/transactions`));
+  }
+
+  async getStorage() {
+      var args = process.argv.slice(3);
```

Comment:
You need to pass this arguments to this function. You shouldn't access arguments directly like this here. 
Please fix this in other places as well.

Reply:
Have fix this as now I am passing arguments to the function.